src/community.rs

```diff
@@ -2,29 +2,45 @@ use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::serde::{Deserialize, Serialize};
 use near_sdk::{env, AccountId};
 
+#[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
+#[serde(crate = "near_sdk::serde")]
+pub struct WikiPage {
+    name: String,
+    content_markdown: String,
+}
+
 #[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
 #[serde(crate = "near_sdk::serde")]
 pub struct Community {
+    pub handle: String,
+    pub admins: Vec<AccountId>,
     pub name: String,
     pub description: String,
-    pub image_url: String,
-    pub thumbnail_url: String,
-    pub overview_page_markdown: String,
-    pub events_page_markdown: String,
-    pub admins: Vec<AccountId>,
-    pub labels: Vec<String>,
-    pub telegram_handles: Vec<String>,
+    pub bio_markdown: Option<String>,
+    pub logo_url: String,
+    pub banner_url: String,
+    pub tag: String,
+    pub github_handle: Option<String>,
+    pub telegram_handle: Option<String>,
+    pub twitter_handle: Option<String>,
+    pub website_url: Option<String>,
     /// JSON string of github board configuration
     pub github: Option<String>,
+    pub sponsorship: Option<bool>,
+    pub wiki1: Option<WikiPage>,
+    pub wiki2: Option<WikiPage>,
+    pub overview_id: Option<u8>,
+    pub events_id: Option<u8>,
```

Comment:
Are these still relevant? I think overview post id / event post id are replaced by wiki1 wiki2 content.

Reply:
I thought that it might be useful for backwards compatibility until communities' full migration to the contract, but if we're going to convert those posts to wiki manually, then yes, I'll remove these properties

Reply:
Okay, I've removed these fields. Migration from posts to wiki doesn't appear to be a sophisticated task.